### PR TITLE
DEV-38 Added Linter and Prettier to the new CRA project

### PR DESCRIPTION
**Pull request by KovDimaY**
To see more info check #87

### CODE REVIEW

File: src/components/charts/line.js [hunk 1]
```diff
@@ -62,9 +61,9 @@ export default class Line extends Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
-    $(this.refs.priceTooltip).tooltip();
-    $(this.refs.oscilationTooltip).tooltip();
-    $(this.refs.frequencyTooltip).tooltip();
+    $(this.refs.priceTooltip).tooltip(); // eslint-disable-line no-undef
+    $(this.refs.oscilationTooltip).tooltip(); // eslint-disable-line no-undef
+    $(this.refs.frequencyTooltip).tooltip(); // eslint-disable-line no-undef
```
Comment (aralroca) [inline src/components/charts/line.js]: in .eslintrc.js you can just:

```js
{
  "globals": {
    "$": true
  }
}
```

Comment (KovDimaY) [reply src/components/charts/line.js]: Oh, true! Thank you so much, it is a great improvement! 🙏

Comment (KovDimaY) [reply src/components/charts/line.js]: DONE

File: .eslintcache [hunk 2]
```diff
@@ -0,0 +1 @@
...a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md","JSXOpeningElement","'name' is assigned a value but never used.","'sankey' is assigned a value but never used.","default-case","Expected a default case.","SwitchStatement","missingDefaultCase","'bins' is assigned a value but never used.","react/jsx-no-target-blank","Using target=\"_blank\" without rel=\"noreferrer\" (which implies rel=\"noopener\") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations","noTargetBlankWithoutNoreferrer",{"range":"229","text":"230"},{"range":"231","text":"230"},{"range":"232","text":"230"},{"range":"233","text":"230"},"'TOP' is assigned a value but never used.","'RIGHT' is assigned a value but never used.","'LEFT' is assigned a value but never used.","no-dupe-keys","Duplicate key 'rangeSelector'.","ObjectExpression","unexpected","'count' is assigned a value but never used.","no-useless-concat","Unexpected string concatenation of literals.","BinaryExpression","unexpectedConcat",[481,481]," rel=\"noreferrer\"",[728,728],[906,906],[1131,1131]]
```
Comment (TetianaKlymchuk) [inline .eslintcache]: Do we need this file commited? 🤔

Comment (KovDimaY) [reply .eslintcache]: Good point! Going to add it to .gitignore, thanks! 👍

**Review by TetianaKlymchuk — APPROVED** [review]

**Pull request merged**